apps/docs/src/app/core/component-docs/dialog/dialog-docs.component.html

```diff
@@ -103,6 +103,21 @@
     <fd-dialog-configuration-example></fd-dialog-configuration-example>
 </component-example>
 <code-example [exampleFiles]="customConfiguration"></code-example>
+
+<separator></separator>
+
+<fd-docs-section-title id="autoLabel" componentName="dialog">
```

Comment:
```suggestion
<fd-docs-section-title id="auto-label" componentName="dialog">
```
In [this PR](https://github.com/SAP/fundamental-ngx/pull/6034#issuecomment-883341410), I am converting ids used in `*.docs.component.html` files to kebab-case for consistency, so could you make this change for `autoLabel` as well?